topi/include/topi/transform.h

```diff
@@ -358,5 +358,85 @@ inline Array<Tensor> split_sections(const Tensor& x,
   return split(x, split_indices, axis, name, tag);
 }
 
+/*!
+* \brief Take elements from an array along an axis.
+*
+* \param inputs
+*    0-The input tensor.
+*    1-The indices of the values to extract.
+* \param axis The axis over which to select values.
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the take operation
+*/
+inline Tensor take(const Array<Tensor>& inputs,
```

Comment:
We should always be consistent with numpy API. 

In this case, https://docs.scipy.org/doc/numpy/reference/generated/numpy.take.html, the argument should be a, and indices

Reply:
BTW, current default value of axis is different with numpy's default value, and the behaviors are also different.